lisa/tools/mde.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import os
+import sys
+import json
+import requests
+from pathlib import Path, PurePath
+from typing import Any
+
+from lisa.executable import Tool
+from lisa.executable import CustomScriptBuilder, CustomScript
+#from . import RemoteCopy, Whoami, Curl
+from .remote_copy import RemoteCopy
+from .whoami import Whoami
+
+class MDE(Tool):
```

Comment:
It's used by MDE test suite only, so it can be in the same file with the test suite.